docs/userguide.md

```diff
@@ -53,11 +54,24 @@ The results will appear in the box below. Select your preferred station, then pr
 
 When you're finished, close *Find Stations* to return to the main window.
 
+## Start/Stop streaming a station <a name = "part_start"></a>
+To start streaming a station, simply click on the little square on the bottom left corner of the station's icon. 
+
+The status bar should then indicate the name of the station you're currently playing.
+
+![Play](screenshots/04_Play_a_station.png)
+
+Click on the corresponding square on another station to switch to that station.
+
```

Comment:
Remove this empty line and put a few spaces in front of the next line to create a line break.

Reply:
OK, I've made the changes.